conans/model/options.py

```diff
@@ -199,6 +198,12 @@ def __init__(self, options=None, options_values=None):
                     tokens = k.split(":", 1)
                     if len(tokens) == 2:
                         package, option = tokens
+                        if "/" not in package:
```

Comment:
But if it is a pattern it will work? Having ``pkg*`` will be ok too?
```suggestion
                        if "/" not in package: and "*" not in package:
```